src/ffi.ts

```diff
@@ -0,0 +1,56 @@
+const lib = Deno.dlopen("./webview/libwebview.so", {
```

Comment:
guessing this needs to get changed from Deno.dlopen to `Plug.prepare` instead then? https://github.com/denosaurs/plug

Reply:
Yep